examples/Cargo.toml

```diff
@@ -51,3 +51,7 @@ trusted_information_retrieval_client = { path = "trusted_information_retrieval/c
 # Third party.
 expect = { path = "../third_party/expect" }
 roughenough = { path = "../third_party/roughenough" }
+prost = { path = "../third_party/prost" }
```

Comment:
Do these need to be added to `experimental/Cargo.toml` too?

Reply:
I'm not sure actually. If we add the overrides you couldn't use a different version of prost for experimental stuff, which at some point we might want? Also, nothing in experimental seems to need the vendored version right now.

What do you think?

Reply:
I don't know much about prost or the vendored version of it, nor do I know much about the experimental crate! I guess @conradgrobler can be a better judge of it. I just mentioned it for completeness :-) But this is certainly something that can be fixed later if needed.

Reply:
I don't think we will need it in the experimental crate at the moment. The only code in there is an unfinished attempt at proxying gRPC with TLS over shared memory.